controller/src/test/java/io/pravega/controller/rest/v1/SecureStreamMetaDataTests.java

```diff
@@ -11,40 +11,50 @@
 
 import io.grpc.ServerBuilder;
 import io.pravega.test.common.SecurityConfigDefaults;
-import io.pravega.controller.server.rpc.auth.AuthHandlerManager;
-import io.pravega.controller.server.rpc.auth.StrongPasswordProcessor;
+import io.pravega.controller.server.security.auth.handler.AuthHandlerManager;
+import io.pravega.controller.server.security.auth.StrongPasswordProcessor;
 import io.pravega.controller.server.rpc.grpc.impl.GRPCServerConfigImpl;
 import io.pravega.test.common.TestUtils;
 import javax.ws.rs.client.Invocation;
 import javax.ws.rs.core.HttpHeaders;
 import javax.ws.rs.core.MultivaluedHashMap;
 import javax.ws.rs.core.MultivaluedMap;
-import org.junit.Before;
-
 import java.io.File;
 import java.io.FileWriter;
+import java.util.Arrays;
+import org.junit.Before;
+
+import static io.pravega.controller.auth.AuthFileUtils.addAuthFileEntry;
 
 public class SecureStreamMetaDataTests extends  StreamMetaDataTests {
+
     @Override
     @Before
     public void setup() throws Exception {
-        File file = File.createTempFile("passwd", ".txt");
-
+        File file = File.createTempFile("SecureStreamMetaDataTests", ".txt");
         StrongPasswordProcessor passwordEncryptor = StrongPasswordProcessor.builder().build();
 
         try (FileWriter writer = new FileWriter(file.getAbsolutePath())) {
             String passwd = passwordEncryptor.encryptPassword("1111_aaaa");
 
             // Admin has READ_WRITE permission to everything
-            writer.write("admin:" + passwd + ":*,READ_UPDATE\n");
+            addAuthFileEntry(writer, "admin", passwd, Arrays.asList("prn::*,READ_UPDATE"));
```

Comment:
Just curious, is `prn::*` equivalent to `prn::/*`

Reply:
No, they are different. 
* `prn::*` gives the user access to all resources.
* `prn::/*` gives the user access to all scopes and resources underneath it. It doesn't provide the user access to the root resource `/`, the user cannot create a scope for instance.

Reply:
Got it.  So in order to create a scope, I'd need to either have:  READ_UPDATE on `prn::*` or READ_UPDATE on `prn::/`, right?

Reply:
Yes, that is right.